resources/views/cruds/datagrids/filters/datagrid-filter.blade.php

```diff
@@ -20,9 +20,9 @@
     @if ($activeFilters > 0)
         @if (empty($bookmark) && isset($entityType))
         @can('create', \App\Models\Bookmark::class)
-            <div class="inline-block cursor-pointer btn2 btn-sm btn-primary break-keep" data-toggle="dialog" data-target="datagrid-bookmark" data-url="{{ route('filters.modal_form', [$campaign, $entityType, 'm' => $mode]) }}">
+            <div class="inline-block cursor-pointer btn2 btn-sm btn-primary break-keep" data-toggle="dialog" data-target="datagrid-filters" data-url="{{ route('filters.modal_form', [$campaign, $entityType, 'm' => $mode]) }}">
                 <x-icon class="fa-solid fa-bookmark" />
-                <span class="hidden sm:inline">{{ __('filters.actions.bookmark') }}</span>
+                <span class="hidden sm:inline"  aria-label="{{ __('filters.actions.bookmark') }}">{{ __('filters.actions.bookmark') }}</span>
```

Comment:
Not needed here but on the div since a screen reader can read the text inside a span just fine